.github/workflows/update-version.yaml

```diff
@@ -3,12 +3,15 @@ on:
   repository_dispatch:
     types:
     - update-on-release
+env:
+  CURRENT_STABLE: 2.3
```

Comment:
As discussed, I think I prefer if the dispatcher side send parameter is_master, based on content of version.txt on master branch on that side.